src/npc.cpp

```diff
@@ -596,14 +597,14 @@ void NpcScriptInterface::registerFunctions()
 int NpcScriptInterface::luaActionSay(lua_State* L)
 {
 	// selfSay(words[, target])
-	Npc* npc = tfs::lua::getScriptEnv()->getNpc();
+	auto npc = tfs::lua::getScriptEnv()->getNpc();
```

Comment:
Could this be `const auto`
```suggestion
	const auto npc = tfs::lua::getScriptEnv()->getNpc();
```